src/commands/inserttablecommand.js

```diff
@@ -53,7 +53,11 @@ export default class InsertTableCommand extends Command {
 		const isRoot = firstPosition.parent === firstPosition.root;
 		const insertPosition = isRoot ? Position.createAt( firstPosition ) : Position.createAfter( firstPosition.parent );
 
-		tableUtils.createTable( insertPosition, rows, columns );
+		model.change( writer => {
```

Comment:
I miss a test for an integration with undo – if you'd use `model.change()` to wrap `writer.setSelection()`, then  (I think) you'd create two undo steps. Or at least – the created undo step might've been incorrect.

Reply:
Try to write an integration test that undo is clear. If this test won't verify that `model.change()` needs to wrap both calls, then make it a unit test and check that a single batch was created.